test/jellyfish_web/controllers/component_controller_test.exs

```diff
@@ -38,6 +38,26 @@ defmodule JellyfishWeb.ComponentControllerTest do
              } = json_response(conn, :ok)["data"]
     end
 
```

Comment:
Could we create two, separate `describe` blocks? One for hls tests and another for rtsp tests? Also, let's remove `@component_type` attribute from l:6